stacks/README.md

```diff
@@ -0,0 +1,68 @@
+# Stacks Deployment Guide
+
+## Table of Contents
+
+-   [Prerequisites](#prerequisites)
+-   [Deployment](#deployment)
+-   [Contract Upgrades](#contract-upgrades)
+-   [Contract Interactions](#contract-interactions)
+
+## Prerequisites
+
+A Stacks contracts deployer account can be created as follows. (if you don't have one already)
+
+```bash
+node stacks/generate-wallet.js
+```
+
+Set `STACKS_MNEMONIC="stacksmnemonic..."` in your `.env` file. If you have a private key you can set `STACKS_PRIVATE_KEY` instead.
```

Comment:
```suggestion
Set `MNEMONIC="stacksmnemonic..."` in your `.env` file. If you have a private key you can set `PRIVATE_KEY` instead.
```
can you switch to the standard names to be consistent with other chain scripts?

Reply:
Since this mnemonic and private key is different per chain, in my opinion it is better to have these separately.
When testing the Checklist for example, you need to have both a Stacks Mnemonic and an EVM Mnemonic, and it is much easier if they are separate, you won't have to edit the .env file between commands.